### PR TITLE
HEV Suit Mk.I

### PR DESCRIPTION
**Pull request by ArchieBeepBoop**
## About The Pull Request

Adds the HEV Suit. Sprites ported from https://github.com/HippieStation/HippieStation/pull/12726

image
image

It can be ordered from cargo via a secure RD access crate, similar to how the M.A.S.O.N. rig can be acquired.

**DOES NOT CURRENTLY CONTAINS SPRITES FOR TAURS**
but if you wanna sprite that and add that in, you're more than welcome. 

Digi leg sprites by @TotallyNotADog 

## Why It's Good For The Game

The HEV suit is basically a version of the Research Hardsuit which has reduced bomb defense stats, reduced acid defense stats but increased radiation protection, while adding a measly extra 5 protection against bullets to compensate for the loss of full bomb protection.

Scientists needed something sturdy to handle hazardous environments for a while, especially due to how often the experimentor spills radiation everywhere, so this should do the trick.

## Changelog
:cl:
add: HEV suit
/:cl:

### CODE REVIEW

**Comment by MalricB** [comment]
POG

**Comment by Dahlular** [comment]
100% approved for a merge if you can yell at someone to sprite the digi version at the very least, since y'know.
Furry server.

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 /obj/item/clothing/neck/tie
 	name = "tie"
-	desc = "A neosilk clip-on tie."
+	desc = "Why do we all have to wear these ridiculous ties?"
```
Comment (Dahlular) [inline code/modules/clothing/neck/_neck.dm:20]: squints

**Review by Dahlular — APPROVED** [review]
i cannot wait to be game ended for having all of these sound effects in our files

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (ArchieBeepBoop) [reply code/modules/clothing/neck/_neck.dm:20]: FEEED ME

**Pull request merged**